sentry-ruby/lib/sentry/metrics.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require 'sentry/metrics/metric'
+require 'sentry/metrics/counter_metric'
+require 'sentry/metrics/distribution_metric'
+require 'sentry/metrics/gauge_metric'
+require 'sentry/metrics/set_metric'
+require 'sentry/metrics/aggregator'
+
+module Sentry
+  module Metrics
+    class << self
+      def incr(key, value = 1.0, unit: 'none', tags: {}, timestamp: nil)
```

Comment:
While Python nor the docs are updated (yet), we agreed on calling it `increment()`. Up to you.